scripts/gisaid_to_ena.py

```diff
@@ -68,17 +78,18 @@ def parse_args(args):
 
 examples:
   # convert GISAID spreadsheet in CSV format to ENA in excel format
-  gisaid_to_ena.py --csv gisaid.csv --out ena.xlsx
+  gisaid_to_ena.py --csv gisaid.csv --outfile ena.xlsx --outformat excel
```

Comment:
It doesn't really work without the taxon id. Could possibly check the `covv_type` column and if it's `betacoronavirus` (this is how GISAID do it), then we should set it to SARS-CoV-2 taxonomy automatically?

```suggestion
  gisaid_to_ena.py --csv gisaid.csv --outfile ena.xlsx --outformat excel --taxon 2697049
```